src/log/log_private.h

```diff
@@ -20,6 +20,8 @@
 #define WT_LSN_MSG(lsn, msg) \
     __wt_msg(session, "%s LSN: [%" PRIu32 "][%" PRIu32 "]", (msg), (lsn)->l.file, (lsn)->l.offset)
 
+#define WT_MAX_LSN_STRING 256
```

Comment:
I suggest 32 (keeping it nicely a power of 2 and aligned) since the string is already only a maximum of 21 characters. 256 is otherwise a lot of extra space. Was there a reason to pick a much larger number?

Reply:
My bad, I got confused between bits and bytes and multiplied 32 by 8.